### ISSUE TITLE
fms: FPLN VNAV predictions not taking entered block fuel into account

### ISSUE THREAD

**Issue by BravoMike99** [flybywiresim/aircraft#8639]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-04-24T17:04:51+00:00",
    "ref": "refs/heads/master",
    "sha": "c989ce225b7a6c201346dfc73609ea45eb203ff0",
    "actor": "frankkopp",
    "event_name": "manual",
    "pretty_release_name": "master:c989ce22",
    "version": "a32nx-v0.12.0-dev.c989ce2"
}
```

### Describe the bug

FMS predictions take into account the FOB instead of the block fuel entered on the INIT FUEL PRED page before engine start. This is especially noticeable when refueling. 
![image](https://github.com/flybywiresim/aircraft/assets/119708186/ec0df74c-6ef1-483d-970b-81d489b5bc53)
![image](https://github.com/flybywiresim/aircraft/assets/119708186/a830ee7a-fa49-46b5-9a23-fc7660a7ecbf)
![image](https://github.com/flybywiresim/aircraft/assets/119708186/23c0a23d-7b88-4e20-83fc-7388521507ef)

### Expected behavior

The block fuel entered on the fuel pred page should be used for predictions prior to engine start. Once the first engine is started, the sensed fuel on board should be used.

### Steps to reproduce

1) Fill INIT A, INIT FUEL PRED & Perf takeoff with any data.
2) Change the fuel on board via the EFB and observe how the EFOB at destination changes on the F-PLN page.

### References (optional)

FCOM DSC-22_20-50-10-25 -> INIT FUEL PRED Page

### Discord Username (optional)

bruno_pt99

**Comment by patmack14** [comment]
Ill get going on this one!

**Comment by CaptCrystal** [comment]
Can this issue be closed since #8664  was merged?

**Comment by BlueberryKing** [comment]
> Can this issue be closed since #8664  was merged? 

Yes, indeed, good catch!
